Automattic/wp-calypso
# Adds Blank Template option and replaces radios with buttons

#### Changes proposed in this Pull Request

* Adds a Blank Template option
* Removes requirement to click a "confirm" button
* Makes Template options a button - click confirms template selection _immediately_

#### Testing instructions

* Create New Page
* See options without "Confirm" buttons
* Click Template option 
* See immediately inserted

Also...

* See Blank Template option
* Click to have a blank Page with no Template

See #33279.
